WooCommerce/Classes/ViewRelated/Products/Edit Product/DefaultProductFormTableViewModel.swift

```diff
@@ -22,19 +28,33 @@ private extension DefaultProductFormTableViewModel {
     }
 
     func primaryFieldRows(product: Product) -> [ProductFormSection.PrimaryFieldRow] {
-        return [
-            .images(product: product),
-            .name(name: product.name),
-            .description(description: product.trimmedFullDescription)
-        ]
+        if canEditImages == false && product.images.isEmpty {
+            return [
+                .name(name: product.name),
+                .description(description: product.trimmedFullDescription)
+            ]
+        } else {
```

Comment:
hmm I generally prefer `if` + `else`, since it tells that these two blocks are in parallel, similar to the switch cases. When the `if` condition is something that's a special case or exception and would return early, I'd use a `guard` statement.

I understand that it could be a personal preference and I could go either way, maybe we can establish a team code style guide. @pmusolino what do think in this case?

Reply:
In this case, I prefer the example proposed by Thuy, just because this case sounds to me like a guard block 👍

Reply:
sorry I somehow missed this comment, I'll update this in a separate PR 🙇🏻‍♀️